app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p><%= t('.greeting', recipient: @email) %></p>
+
+<% if @resource.try(:unconfirmed_email?) %>
+  <p><%= t('.message', email: @resource.unconfirmed_email) %></p>
+<% else %>
+  <p><%= t('.message', email: @resource.email) %></p>
+<% end %>
```

Comment:
Why do you need this view template?  (Will NOT ask the same question on the consequent devise templates that are added to this PR. Please review what files have been added by you running the generator.